packages/rpc-api/src/getBlocks.ts

```diff
@@ -4,10 +4,20 @@ type GetBlocksApiResponse = Slot[];
 
 export type GetBlocksApi = {
     /**
-     * Returns a list of confirmed blocks between two slots
+     * Returns a list of confirmed blocks between two slots (inclusive).
+     *
+     * @see https://solana.com/docs/rpc/http/getblocks
      */
     getBlocks(
+        /** The first slot for which to return a confirmed block */
         startSlot: Slot,
+        /**
+         * The last slot for which to return a confirmed block.
+         *
+         * Must be no more than 500,000 blocks higher than the start slot.
+         *
+         * @defaultValue If not provided, defaults to the latest confirmed slot.
+         */
         endSlotInclusive?: Slot,
```

Comment:
Minor but if both are inclusive, should we either rename to `startSlot` and `endSlot`, or `startSlotInclusive` and `endSlotInclusive`?

Reply:
![Added via Giphy](https://media2.giphy.com/media/h4fHhBZh9m3BecrV0T/giphy.gif?cid=c9487743qv9lxu4vxyzjo79g6bmdk6qc34e6b1ks7wtjgtp5&ep=v1_gifs_search&rid=giphy.gif&ct=g)